Source/SA/RFmxDPD/RFmxDPD.cs

```diff
@@ -0,0 +1,202 @@
+    {
+        #region Type_Definitionss
+        public struct CommonConfiguration
+        {
+            public double DpdMeasurementInterval_s;                                                 
+            public RFmxSpecAnMXDpdMeasurementSampleRateMode DpdSampleRateMode;
+            public double SampleRate_Hz;
+            public RFmxSpecAnMXDpdSignalType SignalType;
+            public RFmxSpecAnMXDpdPreDpdCfrEnabled PreDpdCFREnabled;
+            public RFmxSpecAnMXDpdApplyDpdCfrEnabled ApplyDpdCFREnabled;
+            public double DutAverageInputPower;
+                                                            
+
+            public static CommonConfiguration GetDefault()
+            {
+                return new CommonConfiguration
+                {
+                    DpdMeasurementInterval_s = 100e-6,
+                    DpdSampleRateMode = RFmxSpecAnMXDpdMeasurementSampleRateMode.ReferenceWaveform,
+                    SampleRate_Hz = 320e6,
+                    SignalType = RFmxSpecAnMXDpdSignalType.Modulated,
+                    DutAverageInputPower = -20,
+                    PreDpdCFREnabled = RFmxSpecAnMXDpdPreDpdCfrEnabled.False,
+                    ApplyDpdCFREnabled = RFmxSpecAnMXDpdApplyDpdCfrEnabled.False
+                };
+            }
+
+        }
+        public struct LookupTableConfiguration
+        {
+            public RFmxSpecAnMXDpdLookupTableThresholdType ThresholdType;
+            public RFmxSpecAnMXDpdLookupTableThresholdEnabled ThresholdEnabled;
+            public double ThresholdLevel;
+            public double LookupTableStepSize;
+            public RFmxSpecAnMXDpdApplyDpdLookupTableCorrectionType LookupTableCorrectionType;
+            public RFmxSpecAnMXDpdLookupTableType LookupTableType;
+
+            public static LookupTableConfiguration GetDefault()
+            {
+                return new LookupTableConfiguration
+                {
+                    LookupTableType = RFmxSpecAnMXDpdLookupTableType.Linear,
+                    ThresholdType = RFmxSpecAnMXDpdLookupTableThresholdType.Relative,
+                    ThresholdEnabled = RFmxSpecAnMXDpdLookupTableThresholdEnabled.True,
+                    ThresholdLevel = -20,
+                    LookupTableStepSize = 0.1,
+                    LookupTableCorrectionType = RFmxSpecAnMXDpdApplyDpdLookupTableCorrectionType.MagnitudeAndPhase
+
+                };
+            }
+        }
+
+        public struct MemoryPolynomialConfiguration
+        {
+            public RFmxSpecAnMXDpdApplyDpdMemoryModelCorrectionType MemoryModelCorrectionType;
+            public RFmxSpecAnMXDpdIterativeDpdEnabled IterativeDpdEnabled;
+            public int NumberOfIteration;
+            public int MemoryPolynomialOrder, MemoryPolynomialDepth, MemoryPolynomialLeadOrder, MemoryPolynomialLagOrder,
+            MemoryPolynomialLeadMemoryDepth, MemoryPolynomialLagMemoryDepth, MemoryPolynomialMaxLead, MemoryPolynomialMaxLag;
```

Comment:
Since this is a MemoryPolynomialConfiguration struct we can get rid of the MemoryPolynomial prefix for a lot of these properties